include/powsybl/iidm/VariantManager.hpp

```diff
@@ -30,13 +30,17 @@ class VariantManager {
 public:
     explicit VariantManager(Network& network);
 
-    VariantManager(const VariantManager& variantManager) = delete;
+    VariantManager(const VariantManager&) = delete;
```

Comment:
I don't know if it is clang-tidy compliant (every parameter should be named), I suppose it is. In such case, I think we should be consistent and also remove the parameter of the move constructor.